src/test/java/com/atlassian/db/replica/it/DualConnectionIT.java

```diff
@@ -43,45 +47,48 @@ public void shouldPreserveAutoCommitModeWhileSwitchingFromReplicaToMain() throws
     }
 
     @Test
-    public void shouldRunNextValOnMainDatabase() throws SQLException {
```

Comment:
This mixes up the diff a little. Can we avoid unnecessary "minus lines" in the diff?

Reply:
I'll provide feedback faster I promise